translations/translate.js

```diff
@@ -1385,5 +1389,21 @@ translate = {
         }
 
         return values;
-    } // End overrideValues
-} // End of translate
+    }, // End overrideValues
+
+    getMaxAsString: function(value1, value2) {
+        var maxVal;
+        var val1 = parseInt(value1);
+        var val2 = parseInt(value2);
```

Comment:
Does parseInt throw an exception?  Or just return null?
Might wan to add a test for this condition.

Reply:
If the first character cannot be converted to a number, parseInt() returns NaN. My if statement check of '!val1' will handle checking for NaN